packages/service-library/src/servicelib/rabbitmq/_client.py

```diff
@@ -20,6 +20,15 @@
 
 _DEFAULT_PREFETCH_VALUE: Final[int] = 10
 _DEFAULT_RABBITMQ_EXECUTION_TIMEOUT_S: Final[int] = 5
+_HEADER_X_RETRY_COUNT: Final[str] = "x-retry-count"
+
+
+def _get_unhandled_exception_message(extra_message: str = "") -> str:
```

Comment:
Why not simply using constant as e.g. `UNHANDLED_EXCEPTION_MSG` and use it either as `UNHANDLED_EXCEPTION_MSG + extra_message` or `UNHANDLED_EXCEPTION_MSG.format(extra_message)` ! 

IMO that is much more clear than creating custom functions for something so standard

Reply:
this was changed